charts/oidc-proxy/templates/configmap.yaml

```diff
@@ -21,7 +21,14 @@ data:
 
     Header unset X-Frame-Options
     Header always set X-Frame-Options SAMEORIGIN
-    Header always set Access-Control-Allow-Origin *
+    <If "-n '${CORS_REGEX}'">
+        SetEnvIf Origin ${CORS_REGEX} ORIGIN_SUB_DOMAIN=$0
+        Header add Access-Control-Allow-Origin "%{ORIGIN_SUB_DOMAIN}e" env=ORIGIN_SUB_DOMAIN
```

Comment:
Just curious, why does this use `add` instead of `set`?

Reply:
Per https://stackoverflow.com/questions/1653308/access-control-allow-origin-multiple-origin-domains , it seems like `add` is recommended.